docs/overview.rst

```diff
@@ -10,6 +10,32 @@ abilities. It is meant to be **easy to use** and **easy to self-host**.
 *Kinto* is used at Mozilla and released under the Apache v2 licence.
 
 
+.. _why:
+
+Why
+===
+
+We believe data belongs to the users, and not to the application authors. When
+writing an application, data should be made available to any device, connected
+or offline, and everything should be in sync.
+
+Rather than spending a non-trivial amount of time and expertise on implementing
+all that (and then maintaining it!), you could use Kinto, that does all that
+for you:
+
+- Expose your data over an HTTP interface, unlike a database like PostreSQL
+- Use simple HTTP queries, instead of SQL
+- Use Kinto.js to easily implement offline first clients
+- Choose the database you want from those that Kinto supports, and use a
+  unified API to access its data
+- Manage your data using the handy admin UI
+- Easily set up live push notifications for live synchronisation
+- Make it possible to share data between users (using fine-grained permissions)
+- Take advantage of schema validation if you need it
+- Use tools and libraries like Kinto.js that implements encryption so you don't
```

Comment:
We probably should just remove this right now, and fix it in Kinto.js, so we can support encryption there :)